README.md

```diff
@@ -6,13 +6,101 @@
 
 This extension aims to support the creation of **Liquid templates** with the [Silverfin Templating Language](https://developer.silverfin.com/docs) and the development of [Liquid testing YAML](https://developer.silverfin.com/docs/liquid-testing) files.
 
+## Setup & Basic Usage
+
+### 3rd Party Prerequisites
+- YAML extension: To apply our Schema to YAML files, we need to have [Red Hat's YAML extension](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml) activated (this extension should be added automatically since it is set as a dependency).
+- Auto Close Tag: To enable VS Code to automatically close tags in Liquid files, we need to have [Jun Han's Auto Close Tag](https://marketplace.visualstudio.com/items?itemName=formulahendry.auto-close-tag) activated (this extension should be added automatically since it is set as a dependency).
+  
+### Install Silverfin Extension
+The [Silverfin Development Toolkit extension](https://marketplace.visualstudio.com/items?itemName=Silverfin.silverfin-development-toolkit) can be found in the VS Code Marketplace (you will need to have the version 1.0.0 or higher)
+
+![image](resources/download-extension-1.png)
+
+![image](resources/download-extension-2.png)
+
+### Extension Authorisation
+
+The next stage is to authenticate your device with the Silverfin API so the Silverfin extension can interact with the Silverfin Platform. For this we need to add credentials on our local machine. 
```

Comment:
I elaborated a bit on which credentials the user needs to request from Silverfin.